shuup/front/apps/registration/settings.py

```diff
@@ -9,3 +9,6 @@
 #: This corresponds to using the `default` or `simple`
 #: `django-registration` backends.
 SHUUP_REGISTRATION_REQUIRES_ACTIVATION = True
+
+#: The default registration form
+REGISTRATION_FORM = "shuup.front.apps.registration.forms.PersonRegistrationForm"
```

Comment:
Where is this used? Shouldn't this setting live in registration app? Could the documentation for this setting be better?